CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ### 0.6.0
 
+- New plugins
+    - Falcon Plugin (#146)
+
```

Comment:
You don't need this in this repo

```suggestion
```

Reply:
what? this file don't need manual updates？

Reply:
> what? this file don't need manual updates？

It needs but it is usually updated when we release a new version, anyway since you have already updated it, it's OK to leave it as is